capstone/src/main/java/com/google/univiz/SuggestionDataApi.java

```diff
@@ -0,0 +1,15 @@
+package com.google.univiz;
+
+import java.util.List;
+
+/**
+ * Communicates directly with the data source to get a list of college suggestions related to a
+ * client query
+ */
+public interface SuggestionDataApi {
```

Comment:
I'm confused, how is this interface different from [SearchResource](https://github.com/googleinterns/univiz/pull/57/files#diff-32b6eaa160ada5e3baf248f4e32d1a20R9) that you're working on in PR #57?

Reply:
SearchResource is about translating the Generated Object data into a usable data form the server can use. SuggestionAPI talks directly to the REST API and gets the data from the source

Reply:
In that case how is this interface different from [CollegeDataApi](https://github.com/googleinterns/univiz/blob/master/capstone/src/main/java/com/google/univiz/CollegeDataApi.java) class?

Reply:
OK, after chatting with @mmurray22 offline, i now understand what is happening here. Looks like we have a parallel pipeline for auto complete feature. I'm not a big fan of this design but that is just my subjective opinion. Perhaps the implementation of these two interfaces would share some code.

Also both of these interfaces are right now in the root package while i thought they should be in the scorecard package. But again it's something we can come back and refactor later.

Reply:
I can definitely put this in its own package. I was thinking once this pipeline is done, then we could think about making the two more cohesive. Let me know if you have any more questions!